chore(docs): update `definitions.json` generation docs (#2947)
XRPLF/xrpl.js: CONTRIBUTING.md

This updates `docs/` at the top level, where GitHub Pages looks for the docs.

## Updating `definitions.json`

This should almost always be done using [this script](./packages/ripple-binary-codec/tools/generateDefinitions.js) - if the output needs manual intervention afterwards, consider updating the script instead.

1. Clone / pull the latest changes from [rippled](https://github.com/XRPLF/rippled) - Specifically the `develop` branch is usually the right one.

> 2. Run `node packages/ripple-binary-codec/tools/generateDefinitions.js path/to/rippled` (assuming you're calling this file from the directory this file is in).

Reviewer: NIT: We can update this statement to (assuming you're calling this file from the root directory of xrpl.js) as the Run command specific absolute path.

(assuming you're calling this file from the directory this file is in).

Reviewer: ```suggestion
2. Run `node packages/ripple-binary-codec/tools/generateDefinitions.js path/to/rippled` (assuming you're calling this file from the root directory of xrpl.js).
```

> 2. Run `node packages/ripple-binary-codec/tools/generateDefinitions.js path/to/rippled` (assuming you're calling this file from the root directory of xrpl.js).

Reviewer 2: ```suggestion
2. Run `npm run generateDefinitions path/to/rippled` (assuming you're calling this file from the root directory of xrpl.js).
```

Reviewer 2: nit ^

Author: As we discussed previously, that doesn't work.

Reviewer 2: The npm script works for me. You need to be inside the `packages/ripple-binary-codec` directory.

> 5. Verify that the changes make sense by inspection before submitting, as there may be updates required for the tool depending on the latest amendments we're updating to match.

Reviewer: ```suggestion
3. Verify that the changes make sense by inspection before submitting, as there may be updates required for the tool depending on the latest amendments we're updating to match.
```
